build/OutputDirectories.props

```diff
@@ -25,5 +25,15 @@
       <TestPackagesDir>$(TestOutputDir)/packages/</TestPackagesDir>
       <TestArtifactsDir>$(TestOutputDir)/artifacts/</TestArtifactsDir>
       <TestResultXmlDir>$(TestOutputDir)/results/</TestResultXmlDir>
+      <ArchiveOutputDirectory>$(PackagesDirectory)</ArchiveOutputDirectory>
```

Comment:
I'm starting to see this work as being degrading the cohesiveness of our build code.  If I have a property that is only used in one .targets file, pulling that property into a global .props file is spreading out the code and making it harder to reason about our build logic.